docs/environment.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
 - nbformat
 - jupyter_client
 - ipython
-- sphinx>=1.3.6
+- sphinx>=1.3.6,!=1.5.0,!=1.5.1
```

Comment:
https://github.com/spatialaudio/nbsphinx/pull/85 should also take care of this, it works on both 1.5.0 and 1.5.1 (so long as we have docutils < 1.12 which we will inherit from nbsphinx (we could make this an explicit dependency if we want to)).

But also, the change is already merged on stable for sphinx, so I expect that it will be in 1.5.2. The sphinx maintainers just forgot to port the PR from master to stable before releasing. That's why I was confident that it would be fixed, because it already was fixed, just not merged in the right place at the time of release.